modules/ROOT/pages/dw-core-functions-minusminus.adoc

```diff
@@ -8,19 +8,21 @@ endif::[]
 [[minusminus1]]
 == &#45;&#45;&#40;Array<S&#62;, Array<Any&#62;&#41;: Array<S&#62;
 
-Removes specified items from a list (an array).
+Removes specified items from an array.
 
 
 |===
 | Name   | Description
 
-| `source` | The list (an `Array` type).
-| `toRemove` | Items to remove from the list.
+| `source` | The source array whose items might be removed (an `Array` type). Can have repeating elements. 
+| `toRemove` | The array of items from which to remove from the source array. Need not have repeating elements. 
```

Comment:
@ethanport Hi Ethan, Could you clarify the sentence "Need not have repeating elements"? Did you mean the following?
_The array of items to remove from the source array, can not have repeating elements._